pipelines/concourse.yml

```diff
@@ -1679,3 +1759,40 @@ resources:
     owner: concourse
     repository: concourse
     access_token: ((concourse_github_release.access_token))
+
+- name: resource-types-alpine-image
+  type: registry-image
+  icon: &image-icon docker
+  source:
+    repository: concourse/resource-types
+    tag: alpine
+    username: ((docker.username))
+    password: ((docker.password))
+
+- name: resource-types-ubuntu-image
+  type: registry-image
+  icon: &image-icon docker
+  source:
+    repository: concourse/resource-types
+    tag: ubuntu
+    username: ((docker.username))
+    password: ((docker.password))
+
+- name: concourse-rc-image-alpine
+  type: registry-image
+  icon: *image-icon
+  source:
+    repository: concourse/concourse-rc
+    tag: switch-base
```

Comment:
oops, this should actually be `latest` or just nothing (as `alpine`-based concourse images would still be the default)